docs/index.html

```diff
@@ -148,7 +209,7 @@ <h4>
             </span>
           </p>
           <div class="cde" id='imgCode'>
-            &lt;img src="https://cdn.jsdelivr.net/gh/devicons/devicon/icons/{{selectedIcon.name}}/{{selectedIcon.name}}-{{selectedIcon.svg[selectedSvgIndex]}}.svg" /&gt;<br />
+            &lt;img src="https://cdn.jsdelivr.net/gh/devicons/devicon@latest/icons/{{selectedIcon.name}}/{{selectedIcon.name}}-{{selectedIcon.svg[selectedSvgIndex]}}.svg" /&gt;<br />
```

Comment:
Same here, why not use `<` and `>`? Also why use `@latest`, does it provide any benefit?

Reply:
Already commented above about the same.